### ISSUE TITLE
[WTexScan] Potential NPE in Sidedef Scan

### ISSUE THREAD

**Issue by MTrop** [MTrop/DoomTools#92]
DoomWorld user **Matacrat** found an NPE. Something with GL lumps?

```
java.lang.NullPointerException
        at net.mtrop.doom.tools.WTexScanMain$Context.inspectSidedefs(WTexScanMain.java:425)
        at net.mtrop.doom.tools.WTexScanMain$Context.inspectMap(WTexScanMain.java:329)
        at net.mtrop.doom.tools.WTexScanMain$Context.inspectWAD(WTexScanMain.java:269)
        at net.mtrop.doom.tools.WTexScanMain$Context.processWAD(WTexScanMain.java:259)
        at net.mtrop.doom.tools.WTexScanMain$Context.call(WTexScanMain.java:524)
        at net.mtrop.doom.tools.WTexScanMain$Context.call(WTexScanMain.java:192)
        at net.mtrop.doom.tools.WTexScanMain.call(WTexScanMain.java:666)
        at net.mtrop.doom.tools.doommake.functions.ToolInvocationFunctions$12.execute(ToolInvocationFunctions.java:952)
        at com.blackrook.rookscript.lang.ScriptCommandType$5.execute(Unknown Source)
        at com.blackrook.rookscript.lang.ScriptCommand.execute(Unknown Source)
        at com.blackrook.rookscript.ScriptInstance.step(Unknown Source)
        at com.blackrook.rookscript.ScriptInstance.update(Unknown Source)
        at com.blackrook.rookscript.ScriptInstance.call(Unknown Source)
        at net.mtrop.doom.tools.WadScriptMain$Context.call(WadScriptMain.java:1173)
        at net.mtrop.doom.tools.WadScriptMain$Context.call(WadScriptMain.java:894)
        at net.mtrop.doom.tools.WadScriptMain.call(WadScriptMain.java:1526)
        at net.mtrop.doom.tools.DoomMakeMain$Context.executeTarget(DoomMakeMain.java:606)
        at net.mtrop.doom.tools.DoomMakeMain$Context.call(DoomMakeMain.java:455)
        at net.mtrop.doom.tools.DoomMakeMain$Context.call(DoomMakeMain.java:303)
        at net.mtrop.doom.tools.DoomMakeMain.call(DoomMakeMain.java:982)
        at net.mtrop.doom.tools.DoomMakeMain.main(DoomMakeMain.java:1003)
ERROR: [ToolError]: WTexScan threw an error.
```

**Comment by MTrop** [comment]
Uh oh - it looks like DoomStruct is detecting `GL_MAP01` as a map header (should be considered part of the map).

```
# Inspecting C:\Users\Matt\Desktop\orange_rc2.wad...
#    Opening map MAP01...
#    Format is DOOM...
#        Reading SIDEDEFS...
#        Reading SECTORS...
#    Opening map GL_MAP01...
#    Format is DOOM...
#        Reading SIDEDEFS...
java.lang.NullPointerException
	at net.mtrop.doom.tools.WTexScanMain$Context.inspectSidedefs(WTexScanMain.java:425)
	at net.mtrop.doom.tools.WTexScanMain$Context.inspectMap(WTexScanMain.java:329)
...
```

**Comment by MTrop** [comment]
Added issue to DoomStruct: https://github.com/MTrop/DoomStruct/issues/16

**Comment by MTrop** [comment]
Fixed in commit 0730328b1c3dae82b8016e478e6507305715fc46.
